django_fido/backends.py

```diff
@@ -1,51 +1,60 @@
 """Authentication backends for Django FIDO."""
 from __future__ import unicode_literals
 
+import base64
 import logging
+from typing import Any, Dict, Optional
 
+from cryptography.exceptions import InvalidSignature
 from django.contrib import messages
 from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
 from django.core.exceptions import PermissionDenied
+from django.http import HttpRequest
 from django.utils.translation import ugettext_lazy as _
-from u2flib_server import u2f
+from fido2.server import Fido2Server
 
 _LOGGER = logging.getLogger(__name__)
 
 
-class U2fAuthenticationBackend(object):
+class Fido2AuthenticationBackend(object):
```

Comment:
Probably no reason to inherit from an `object` since this is Py3 only.

This applies to all classes.

Reply:
I'll rather fix that separately.

Reply:
Ok.